Prox/Prox/Utilities/EventNotificationsManager.swift

```diff
@@ -6,8 +6,20 @@ import Foundation
 import CoreLocation
 import UserNotifications
 
+private let sentNotificationDictKey = "sent_notifications_dict"
+
 class EventNotificationsManager {
 
+    fileprivate var sentNotifications: [String: [String]] {
+        get {
+            return UserDefaults.standard.dictionary(forKey: sentNotificationDictKey) as? [String : [String]] ?? [String : [String]]()
```

Comment:
We probably don't need to care but I'd expect this to grow infinitely, taking up a lot of disk space.

Perhaps add a TODO so we don't forget?